tdmpc2/common/buffer.py

```diff
@@ -92,5 +92,5 @@ def add(self, td):
 
 	def sample(self):
 		"""Sample a batch of subsequences from the buffer."""
-		td = self._buffer.sample().view(-1, self.cfg.horizon+1).permute(1, 0)
+		td = self._buffer.sample().view(-1, self.cfg.horizon+1).permute(1, 0).clone()
```

Comment:
because we're doing views but we'll feed this to the model, better cloning the values (to make sure we don't have a view on CUDA)
This could be handled by CudaGraphModule btw! I will patch it